docs/coding-guidelines/css.md

```diff
@@ -1,5 +1,17 @@
 # CSS/Sass Coding Guidelines
 
+## Principles
+
+We're transitioning to a component-based design system. When styling, adhere to the following principles:
+
+ - Rely on the Design System components, props and variants as much as possible.
```

Comment:
Without a link to the design system reference site, this might raise more questions than answers. 😓

Reply:
Should I rename it to `@wordpress/components` and `@automattic/components` for now?

Reply:
Could do that, yes. Or, can include both - "design system", with some clarification in parentheses, plus a nudge to ask in `#design-systems` or `#components` in Slack.

Reply:
Done :)

Reply:
Thanks Riad!